adafruit_bus_device/i2c_device.py

```diff
@@ -101,6 +101,44 @@ def write(self, buf, **kwargs):
         """
         self.i2c.writeto(self.device_address, buf, **kwargs)
 
+#pylint: disable-msg=too-many-arguments
+    def write_then_readinto(self, out_buffer, in_buffer,
```

Comment:
Add a ` *,` after `in_buffer,` to make the remaining arguments kwarg-only. This way the start and stop numbers must always be "labeled".